docs/faq_tips_tricks.rst

```diff
@@ -126,3 +126,23 @@ A symptomatic output looks like: ::
 
 If you would like to run *fMRIPrep* in parallel on multiple subjects please use
 `this method <https://neurostars.org/t/fmriprep-workaround-for-running-subjects-in-parallel/4449>`__.
+
+
+.. _upgrading:
```

Comment:
I think we need to rename this file to `faq` and update any links to `faq_tips_tricks`. This is to make the `faq.html` link work, but I think it's good to move to a guessable file name anyway.